talk/basicconcepts/functions.tex

```diff
@@ -24,7 +24,7 @@
 
       // no parameter
       void hello() {
-      	std::cout << "Hello World";
+         std::cout << "Hello World";
```

Comment:
```suggestion
        std::cout << "Hello World";
```
I think we use two spaces indentation everywhere else.

Reply:
Very good, I must have typed on space too much.